Firmware/first_lay_cal.cpp

```diff
@@ -96,65 +107,61 @@ bool lay1cal_load_filament(uint8_t filament)
 //! @param extrusion_width the width of the extrusion layer
 void lay1cal_intro_line(bool extraPurgeNeeded, float layer_height, float extrusion_width)
 {
-    static const char cmd_intro_mmu_3[] PROGMEM = "G1 X55 E29 F1073";
-    static const char cmd_intro_mmu_4[] PROGMEM = "G1 X5 E29 F1800";
-    static const char cmd_intro_mmu_5[] PROGMEM = "G1 X55 E8 F2000";
-    static const char cmd_intro_mmu_6[] PROGMEM = "G1 Z0.3 F1000";
-    static const char cmd_intro_mmu_8[] PROGMEM = "G1 X240 E25  F2200";
-    static const char cmd_intro_mmu_9[] PROGMEM = "G1 Y-2 F1000";
-    static const char cmd_intro_mmu_10[] PROGMEM = "G1 X55 E25 F1400";
-    static const char cmd_intro_mmu_11[] PROGMEM = "G1 Z0.2 F1000";
-    static const char cmd_intro_mmu_12[] PROGMEM = "G1 X5 E4 F1000";
```

Comment:
Is `cmd_intro_mmu_12` removal intentional? Also `cmd_intro_mmu_10` is changed, why?

Reply:
Yes both with and without MMU purge line end at ~same position to continue with the new first layer cal pattern directly after the purge to keep the nozzle pressure up.